CI/hf_mirror/download_hf_models.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+df -h  # debug purpose to make sure efs is attached correclty
+git clone https://github.com/autogluon/autogluon.git
```

Comment:
Why we need autogluon repo to be present?

Reply:
We need to fetch the model list dynamically to know which models to download